CodeMaid/Logic/Cleaning/FileHeaderLogic.cs

```diff
@@ -69,32 +85,37 @@ internal void UpdateFileHeader(TextDocument textDocument)
         }
 
         /// <summary>
-        /// Gets the file header from settings based on the language of the specified document.
+        /// Reads the first lines of a document
         /// </summary>
-        /// <param name="textDocument">The text document.</param>
-        /// <returns>A file header from settings.</returns>
-        private static string GetFileHeaderFromSettings(TextDocument textDocument)
+        /// <param name="textDocument">The document to read</param>
+        /// <returns>A string representing the first <see cref="HeaderMaxNbLines"/> lines of the document</returns>
+        private string ReadHeaderBlock(TextDocument textDocument)
+        {
+            var headerNbLines = Math.Min(HeaderMaxNbLines, textDocument.EndPoint.Line);
+            var headerBlockEnd = textDocument.StartPoint.CreateEditPoint();
+            
+            headerBlockEnd.MoveToLineAndOffset(headerNbLines, 1);
+
+            var headerBlockStart = textDocument.StartPoint.CreateEditPoint();
+            var headerBlock = headerBlockStart.GetText(headerBlockEnd);
+
+            return headerBlock;
```

Comment:
There is a `GetLines` method that will simplify this logic.  Something like:
```
var point = textDocument.StartPoint.CreateEditPoint();
var text = point.GetLines(1, headerNbLines);
```

Reply:
you're right, fixed